src/Helpers/Configuration.test.ts

```diff
@@ -0,0 +1,51 @@
+import { describe } from 'riteway'
+import { createEnvToConfigurationKeyMap, toPair } from './Configuration'
```

Comment:
We should not be importing these functions from Configuration, let's move them to a different file, Helpers probably.

Reply:
It's importing this from `Helpers/Configuration`. Linting wouldn't let me do an absolute import. Should I rename the helper file?